src/fontra/backends/fontra.py

```diff
@@ -92,6 +93,13 @@ def glyphInfoPath(self):
     def glyphsDir(self):
         return self.path / self.glyphsDirName
 
+    @property
+    def binaryDataPath(self):
```

Comment:
Please do *not* work on the Fontra backlend until we've actually designed how that data should be stored in there.